### DISCUSSION TITLE
Support for ESP3D printer controllers

### DISCUSSION THREAD

**Discussion by KingBain** [OrcaSlicer/OrcaSlicer#2690] [Ideas]
I noticed that the ESP3D web server & interface wasn't supported by Orca, is there any documentation on that specific section of the slicer software that I could read to learn how to add support for other IP enabled machines ?

I might like to help write a module to support ESP3D 

https://github.com/luc-github/ESP3D

**Comment by PreyK** [comment]
Would love to have this as well, recently added ESP3D to my printer, looking into how I could implement it in orcaslicer, based on the MKS connector it looks pretty straight-forward.
